tools/data/ava_kinetics/extract_rgb_frames.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Can we reuse the script `tools/data/build_rawframes.py` like: https://github.com/open-mmlab/mmaction2/blob/dev-1.x/tools/data/kinetics/extract_rgb_frames.sh?

Reply:
Yes we can. But I did not try that and I am not sure about the the correctness of using this script.